plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -412,3 +429,14 @@ func MapPodsToMachines(lister algorithm.PodLister) (map[string][]*api.Pod, error
 	}
 	return machineToPods, nil
 }
+
+func haveSame(a1, a2 []string) bool {
```

Comment:
right, that's what I read in the code.  Maybe a comment to clarify that this is the intent and why it's ok to return true after 1 common value?  Can the rest of the values differ and that's ok?